ocs_ci/ocs/node.py

```diff
@@ -2350,3 +2350,26 @@ def wait_for_new_worker_node_ipi(machineset, old_wnodes, timeout=900):
     wait_for_nodes_status([new_wnode.name])
     log.info(f"The new worker node {new_wnode.name} is in a Ready state!")
     return new_wnode
+
+
+def check_for_zombie_process_on_node(node_name=None):
+    """
+    Check if there are any zombie process on the nodes
+
+    Args:
+        node_name (list): Node names list to check for zombie process
+
+    Raise:
+        ZombieProcessFoundException: In case zombie process are found on the node
+
+    """
+    node_obj_list = get_node_objs(node_name) if node_name else get_node_objs()
+    for node_obj in node_obj_list:
+        debug_cmd = f'debug nodes/{node_obj.name} -- chroot /host /bin/bash -c "top -b1 -n1 | grep Z"'
```

Comment:
Only 'Z'?
I think there could be a process containing the letter 'Z' like FileZilla that is not a zombie.
WDYT?

Reply:
Yes, there can be cases like that. I changed the implementation to use ps -A -ostat,pid,ppid | grep -e "[zZ]